src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx

```diff
@@ -19,92 +19,87 @@ import aiExplainThisErrorPopUp from 'images/ai_screenshot-crop_explain-this-erro
 
 import aiAskAiinErrorsInbox from 'images/ai_screenshot-crop_ask-ai-in-errors-inbox.webp'
 
-New Relic AI (NRAI) is designed to help every member of your team improve observability, not just engineers. We combined OpenAI's large language models (LLMs) with our unified telemetry data platform to create an AI assistant that deeply understands how both your system and New Relic function. New Relic AI helps you demystify the complexity of your stack, solve service outages, and improve your efficiency simply by letting you ask questions in plain language. 
+It combines OpenAI's large language models (LLMs) with New Relic's data platform to understand both your system and the New Relic platform itself, providing deeper insights and faster solutions. New Relic AI helps you demystify the complexity of your stack, solve service outages, and improve your efficiency simply by letting you ask questions in plain language. 
 
-## Ways to interact with New Relic AI [#work-with-NRAI]
+Ask New Relic AI to help you:
+* [Analyze error logs](https://docs.newrelic.com/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai/#errors)
+* [Dive deeper in stack traces](https://docs.newrelic.com/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai/#stack-traces)
+* [Identify gaps in your alerting ecosystem](https://docs.newrelic.com/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai/#coverage-gaps)
+* [Create synthetic monitors](https://docs.newrelic.com/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai/#)
+* [Understand your dashboards](https://docs.newrelic.com/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai/#dashboards)
+* [Check the status of support tickets](https://docs.newrelic.com/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai/#support)
 
-We help you get to the heart of your system by offering quick and easy ways to ask questions: AI chat, and the NRQL console. 
+<Video
+ type="youtube"
+ id="7vBhF4Ph_LA?si=UyaTG9LwSfkqVUU5"
+/>
 
-### Ask AI questions [#ask-AI]
+## Ways to interact with New Relic AI [#work-with-NRAI]
```

Comment:
@homelessbirds The right-hand nav doesn't seem to show the H3s, so I would remove this and make all the subsequent H3s into H2s

Reply:
Fixed!